dap_aria_mapping/pipeline/entity_processing/reformat_entities.py

```diff
@@ -0,0 +1,87 @@
+from typing import Dict
+import boto3
+import json
+
+from metaflow import FlowSpec, step, Parameter, batch
+
+
+PATENT_ANNOTATION_OUTPUT = "inputs/data_collection/patents/preprocessed_annotated_patents.json"
+OPENALEX_ANNOTATION_OUTPUT = "inputs/data_collection/processed_openalex/preprocessed_annotated_abstracts.json"
```

Comment:
been re-running this flow with a slight change because these files currently are currently the same output!

Reply:
sorry its taking me a long time! hopefully will have a validation sample to label tomorrow